src/execution/operator/csv_scanner/scanner/string_value_scanner.cpp

```diff
@@ -958,8 +962,14 @@ void StringValueScanner::Flush(DataChunk &insert_chunk) {
 					auto borked_line = result.line_positions_per_row[line_error].ReconstructCurrentLine(
 					    first_nl, result.buffer_handles, result.PrintErrorLine());
 					std::ostringstream error;
-					error << "Could not convert string \"" << parse_vector.GetValue(line_error) << "\" to \'"
-					      << LogicalTypeIdToString(type.id()) << "\'";
+					if (type.IsJSONType()) {
+						error << "Could not convert string \"" << parse_vector.GetValue(line_error) << "\" to \'"
+						      << "JSON"
+						      << "\'";
+					} else {
+						error << "Could not convert string \"" << parse_vector.GetValue(line_error) << "\" to \'"
+						      << LogicalTypeIdToString(type.id()) << "\'";
```

Comment:
Maybe we can use `type.ToString()` here instead of duplicating the error message?